SYCL/cgeist/single_task.cpp

```diff
@@ -0,0 +1,33 @@
+// RUN: %clangxx -fsycl -fsycl-targets=spir64-unknown-unknown-syclmlir %s -o %t.out 2>&1
+// RUN: %CPU_RUN_PLACEHOLDER %t.out
+// RUN: %GPU_RUN_PLACEHOLDER %t.out
+// RUN: %ACC_RUN_PLACEHOLDER %t.out
+// REQUIRES: linux
+
+#include <sycl/sycl.hpp>
+
```

Comment:
If you want the test code to be less verbose, it is common to use a `using namespace sycl` in SYCL code, so you don't have to explicitly qualify all SYCL classes.